### ISSUE TITLE
Make passing "index" to "serve-static" optional

### ISSUE THREAD

**Issue by vysokiylev** [tkoenig89/express-static-gzip#31]
I don't want "changeUrlFromEmptyToIndexHtml" function to change my req.url. The only way to avoid it is to pass `{ index: false } `  option to "expressStaticGzip", but this option is also passed to "ServeStatic". This is not what I expected to do. Please, make passing "index" to "serve-static" or calling "changeUrlFromEmptyToIndexHtml" optional

**Comment by vysokiylev** [comment]
Or return "indexFromEmptyFile" option. I had to downgrade version to 0.3.2 to make it work

**Comment by tkoenig89** [comment]
You have a valid point here, that one would like to deactivate the `index` settings, without passing it's value to `ServeStatic`.

As changing something related to this might have a larger impact on users of this package, this should best be done in a non intrusive manor. 

My first idea would be to add an additional option (which would not break existing behaviour). Maybe something like

```JavaScript
const options = {
  index: false,
  passIndexToServeStatic: false //default is true
}
```

Any thoughts on that?

**Comment by tkoenig89** [comment]
I just released version 2.0, which should solve your issue. Just use the new options syntax:
```JavaScript
const options = {
  index: false,                        // applies to expressStaticGzip only
  serveStatic: {
    index: 'index.html'                // applies to serveStatic only
  }
}
```

**Comment by vysokiylev** [comment]
Thank you, it is much better! I'll close the issue.
